### PR TITLE
W-15435765 Update CIM documentation

### PR DESCRIPTION
**Pull request by kckane-mulesoft**

### CODE REVIEW

File: accelerators-cim/2.0/modules/ROOT/pages/cim-subject-areas.adoc [hunk 1]
```diff
@@ -1,8 +1,12 @@
+:imagesdir: ../assets/images
+
 = CIM Subject Areas
 
 The model content is organized into domains, or Subject Areas, which represent major business concepts. Each Subject Area defines one or more Entity Groups, which in turn consist of a related collection of Entities. Each Entity has a set of Attributes defining the unique characteristics of the Entity.
 
-The following subject areas are currently supported - follow the links or refer to the tabs on the left for more details on each domain.
+== Supported Subject Areas
+
+The following subject areas are currently supported. Each subject area includes an Entity Relationship Diagram (ERD) for each entity group defined under that area. See below for additional details on the conventions used in the diagrams.
```
Comment (tasmoore) [inline accelerators-cim/2.0/modules/ROOT/pages/cim-subject-areas.adoc]:
```suggestion
CIM supports the following subject areas, each of which includes an Entity Relationship Diagram (ERD). See <<diagram_conventions>> for the conventions used in the diagrams.
```

File: accelerators-cim/2.0/modules/ROOT/pages/cim-subject-areas.adoc [hunk 2]
```diff
@@ -21,6 +25,19 @@ The following subject areas are currently supported - follow the links or refer
 | Entities related to technical resources and artifacts.
 |===
 
+== Diagram conventions
```
Comment (tasmoore) [inline accelerators-cim/2.0/modules/ROOT/pages/cim-subject-areas.adoc]:
```suggestion
[[diagram_conventions]]
== Diagram Conventions
```

File: accelerators-cim/2.0/modules/ROOT/pages/cim-subject-areas.adoc [hunk 3]
```diff
@@ -21,6 +25,19 @@ The following subject areas are currently supported - follow the links or refer
...
+
+The diagrams included in each subject group show the entities and relationships defined within that group. In most cases, relationships to entities defined in other subject areas are also provided for context. The following table summarizes the conventions used to indicate the status of each entity.
```
Comment (tasmoore) [inline accelerators-cim/2.0/modules/ROOT/pages/cim-subject-areas.adoc]:
```suggestion
The diagrams included in each subject group show the entities and relationships defined within that group. Relationships to entities defined in other subject areas are also provided for context. The following table summarizes the conventions used to indicate the status of each entity.
```

File: accelerators-cim/2.0/modules/ROOT/pages/cim-subject-areas.adoc [hunk 4]
```diff
@@ -21,6 +25,19 @@ The following subject areas are currently supported - follow the links or refer
...
+
+[%header%autowidth.spread^]
+|===
+| Symbol | Entity Status | Description
+| image:defined-group-entity.png[Defined Group Entity,50%] | Defined Group Entity | An entity in the current group that has a concrete definition
```
Comment (tasmoore) [inline accelerators-cim/2.0/modules/ROOT/pages/cim-subject-areas.adoc]:
```suggestion
| image:defined-group-entity.png[Defined Group Entity,50%] | Defined Group Entity | An entity in the current group that has a concrete definition.
```

File: accelerators-cim/2.0/modules/ROOT/pages/cim-subject-areas.adoc [hunk 5]
```diff
@@ -21,6 +25,19 @@ The following subject areas are currently supported - follow the links or refer
...
+| image:undefined-group-entity.png[Undefined Group Entity,50%] | Undefined Group Entity | An entity belonging to the current group that does not have a concrete definition
+| image:defined-external-entity.png[Defined External Entity,50%] | Defined External Entity | An entity defined in a different entity group or subject area
+| image:undefined-external-entity.png[Undefined External Entity,50%] | Undefined External Entity | An entity belonging to a different group or subject area that has not been defined
```
Comment (tasmoore) [inline accelerators-cim/2.0/modules/ROOT/pages/cim-subject-areas.adoc]:
```suggestion
| image:undefined-external-entity.png[Undefined External Entity,50%] | Undefined External Entity | An entity belonging to a different group or subject area that has not been defined.
```

File: accelerators-cim/2.0/modules/ROOT/pages/cim-subject-areas.adoc [hunk 6]
```diff
@@ -21,6 +25,19 @@ The following subject areas are currently supported - follow the links or refer
...
+== Diagram conventions
+
+The diagrams included in each subject group show the entities and relationships defined within that group. In most cases, relationships to entities defined in other subject areas are also provided for context. The following table summarizes the conventions used to indicate the status of each entity.
+
+[%header%autowidth.spread^]
+|===
+| Symbol | Entity Status | Description
+| image:defined-group-entity.png[Defined Group Entity,50%] | Defined Group Entity | An entity in the current group that has a concrete definition
+| image:undefined-group-entity.png[Undefined Group Entity,50%] | Undefined Group Entity | An entity belonging to the current group that does not have a concrete definition
+| image:defined-external-entity.png[Defined External Entity,50%] | Defined External Entity | An entity defined in a different entity group or subject area
```
Comment (tasmoore) [inline accelerators-cim/2.0/modules/ROOT/pages/cim-subject-areas.adoc]:
```suggestion
| image:defined-external-entity.png[Defined External Entity,50%] | Defined External Entity | An entity defined in a different entity group or subject area.
```

File: accelerators-cim/2.0/modules/ROOT/pages/cim-subject-areas.adoc [hunk 7]
```diff
@@ -21,6 +25,19 @@ The following subject areas are currently supported - follow the links or refer
...
 
+== Diagram conventions
+
+The diagrams included in each subject group show the entities and relationships defined within that group. In most cases, relationships to entities defined in other subject areas are also provided for context. The following table summarizes the conventions used to indicate the status of each entity.
+
+[%header%autowidth.spread^]
+|===
+| Symbol | Entity Status | Description
+| image:defined-group-entity.png[Defined Group Entity,50%] | Defined Group Entity | An entity in the current group that has a concrete definition
+| image:undefined-group-entity.png[Undefined Group Entity,50%] | Undefined Group Entity | An entity belonging to the current group that does not have a concrete definition
```
Comment (tasmoore) [inline accelerators-cim/2.0/modules/ROOT/pages/cim-subject-areas.adoc]:
```suggestion
| image:undefined-group-entity.png[Undefined Group Entity,50%] | Undefined Group Entity | An entity belonging to the current group that does not have a concrete definition.
```

File: retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc [hunk 8]
```diff
@@ -26,7 +26,7 @@ The following objects must be created manually or via Apex script:
 
 === Create Custom Object via Apex Script
 
-The`CustomObjectUtility` and `MetadataService` Apex classes are used to create a custom object programmatically. The source can be found on https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources/[RCG Common Resources] project and https://github.com/financialforcedev/apex-mdapi/blob/master/apex-mdapi/src/classes/MetadataService.cls[MetadataService] respectively. Download the project and follow the steps below.
+The`CustomObjectUtility` and `MetadataService` Apex classes are used to create a custom object programmatically. The source can be found on https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources-src/[RCG Common Resources] project and https://github.com/financialforcedev/apex-mdapi/blob/master/apex-mdapi/src/classes/MetadataService.cls[MetadataService] respectively. Download the project and follow the steps below.
```
Comment (tasmoore) [inline retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc]:
```suggestion
Use the`CustomObjectUtility` and `MetadataService` Apex classes to create a custom object programmatically. The source is on the https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources-src/[RCG Common Resources] project and https://github.com/financialforcedev/apex-mdapi/blob/master/apex-mdapi/src/classes/MetadataService.cls[MetadataService] respectively. Download the project and follow these steps:
```

File: retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc [hunk 9]
```diff
@@ -290,7 +290,7 @@ Unique fields should be case-sensitive.
 
 === Create Fields via Apex Script
 
-Use the Apex class `CustomFieldUtility` to create custom fields programmatically. See the https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources/[RCG Common Resources] project to find the source for this class. Once you have downloaded the project, follow these steps:
+Use the Apex class `CustomFieldUtility` to create custom fields programmatically. See the https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources-src/[RCG Common Resources] project to find the source for this class. Once you have downloaded the project, follow these steps:
```
Comment (tasmoore) [inline retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc]:
```suggestion
Use the Apex class `CustomFieldUtility` to create custom fields programmatically. See the https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources-src/[RCG Common Resources] project to find the source for this class. Download the project and follow these steps:
```

File: retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc [hunk 10]
```diff
@@ -406,7 +406,7 @@ If the option to reset your security token is not available, it is likely that o
 
 == Create Platform Events and Apex Triggers
 
-The following Platform Event Object and Apex Triggers must be created in order to capture updates from Salesforce. The scripts to create these Apex Triggers and add fields to Platform Event Object are located in the https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources/[RCG Common Resources] project.
+The following Platform Event Object and Apex Triggers must be created in order to capture updates from Salesforce. The scripts to create these Apex Triggers and add fields to Platform Event Object are located in the https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources-src/[RCG Common Resources] project.
```
Comment (tasmoore) [inline retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc:409]:
```suggestion
To capture updates from Salesforce, create the following Platform Event Object and Apex Triggers. The scripts to create the Apex Triggers and add fields to the Platform Event Object are in the https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources-src/[RCG Common Resources] project.
```

File: retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc [hunk 11]
```diff
@@ -426,7 +426,7 @@ This implementation requires `Platform Event` to be created in Salesforce. Follo
 . Click the *New Platform Event* button to create an Object. Complete the *Platform Information Details* fields, including *Label*, *Plural Label*, *Object with Salesforce Object*, and *Salesforce Platform Event API Name* as indicated in the previous table.
 . Click *Save*.
 
-. To add Custom Fields & RelationShips to Platform Event Definitions, run the utility script `CreatePlatformEventsCustomFields.txt` available in the `rcg-common-resources/salesforce` folder of https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources/[RCG Common Resources] project.
+. To add Custom Fields & RelationShips to Platform Event Definitions, run the utility script `CreatePlatformEventsCustomFields.txt` available in the `rcg-common-resources/salesforce` folder of https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources-src/[RCG Common Resources] project.
```
Comment (tasmoore) [inline retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc]:
```suggestion
. To add Custom Fields & RelationShips to Platform Event Definitions, run the utility script `CreatePlatformEventsCustomFields.txt`, which is in the `rcg-common-resources/salesforce` folder of the https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources-src/[RCG Common Resources] project.
```

File: retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc [hunk 12]
```diff
@@ -438,7 +438,7 @@ Follow these steps to create the Apex Triggers:
 . Search for the Salesforce object. This opens a page that shows the details of the object.
 . Click *Triggers* in the Details section to view a list of available triggers for the object.
 . Click *New*.
-. Search for the Apex Trigger name, which is indicated in the previous table. Copy the Trigger Script code `CreateApexTriggers.txt` available in the `rcg-common-resources/salesforce` folder of https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources/[RCG Common Resources] project and paste it into the script editor.
+. Search for the Apex Trigger name, which is indicated in the previous table. Copy the Trigger Script code `CreateApexTriggers.txt` available in the `rcg-common-resources/salesforce` folder of https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources-src/[RCG Common Resources] project and paste it into the script editor.
```
Comment (tasmoore) [inline retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc]:
```suggestion
. Search for the Apex Trigger name, which is indicated in the previous table. Copy the Trigger Script code `CreateApexTriggers.txt` from the `rcg-common-resources/salesforce` folder of the https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources-src/[RCG Common Resources] project and paste it into the script editor.
```

**Review by tasmoore — DISMISSED** [review]

File: retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc (hunk 8 above)
Comment (kckane-mulesoft) [reply retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc]: This is one of those changes that needs to be back-ported to beta.

File: retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc (hunk 9 above)
Comment (kckane-mulesoft) [reply retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc]: This is one of those changes that needs to be back-ported to beta.

File: retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc (hunk 11 above)
Comment (kckane-mulesoft) [reply retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc]: This is one of those changes that needs to be back-ported to beta.

File: retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc (hunk 12 above)
Comment (kckane-mulesoft) [reply retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc]: This is one of those changes that needs to be back-ported to beta.

**Review by tasmoore — APPROVED** [review]

**Pull request merged**